doc/example/fixed_parameters.ipynb

```diff
@@ -17,6 +17,16 @@
     "of them to specified values."
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# install if not done yet\n",
+    "# !pip install pypesto --quiet"
```

Comment:
```suggestion
    "# %pip install pypesto --quiet"
```
To ensure the correct pip is used (https://ipython.readthedocs.io/en/stable/interactive/magics.html#magic-pip).